recipes/base_install.rb

```diff
@@ -186,7 +186,9 @@
 include_recipe "aws-parallelcluster::_lustre_install"
 
 # Install EFA & Intel MPI
-if (node['platform'] == 'centos' && node['platform_version'].to_i >= 7) || node['platform'] == 'amazon' || (node['platform'] == 'ubuntu' && node['platform_version'] == "16.04")
+if (node['platform'] == 'centos' && node['platform_version'].to_i >= 7) || node['platform'] == 'amazon' \
+  || (node['platform'] == 'ubuntu' && node['platform_version'] == "16.04") \
+  || (node['platform'] == 'ubuntu' && node['platform_version'] == "18.04")
```

Comment:
minor, is it possible to put the check `node['platform'] == 'ubuntu' ` in common?
or use something like we use for centos `>= 7`

Reply:
Something like node['platform_version'] >= 16.04 you mean? Yes I like it

Reply:
done